lib/dialects/mysql/schema/compiler.js

```diff
@@ -3,7 +3,7 @@
 const inherits = require('inherits');
 const SchemaCompiler = require('../../../schema/compiler');
 
-const { some } = require('lodash');
+const some = require('lodash/some');
```

Comment:
Should be possible to replace with native `some`

Reply:
this place and next - lodash for many functions accept also object

Reply:
I was not certain whether `resp` is always an array, or if it could be `undefined` in some situations. `_.some(undefined, ...)` returns `false` while `undefined.some` throws.